pkg/backend/display/testdata/not-truncated/failed-stacks-dont-rewrite-operations.json.interactive-100x80-cooked.stdout.txt

```diff
@@ -1,38 +1,10 @@
 <{%fg 13%}><{%bold%}>View in Browser (Ctrl+O): <{%underline%}><{%fg 12%}>link<{%reset%}>
 
-<%hide-cursor%>
     <{%underline%}><{%fg 12%}>Type<{%reset%}>                 <{%underline%}><{%fg 12%}>Name<{%reset%}>           <{%underline%}><{%fg 12%}>Status<{%reset%}>     <{%underline%}><{%fg 12%}>Info<{%reset%}><%clear-to-end%>
```

Comment:
This is not possible to review, so I will just ask how were these files generated? What is the true change here?

Reply:
I think the question behind the question is: how should you review this?

These files are generated via `progress_test.go`, and we have this pattern throughout the pulumi/pulumi codebase as we treat the text output of the CLI as part of an API contract, albeit one consumed by our users. Faithfully recording the whole output in these test files is the point, and this PR fixes the test to record what the renderer does today.

As we're only fixing the test to generate the correct snapshots, _not_ changing the rendering code. So if there were any defects in these snapshots of the rendered output we would want to fix that in a second PR, to separate "fixing the test" from "fixing the output". In that second PR, the diff would hopefully show a tactical change to the output that represents what the real change is in rendered text. For example, if we had a bug in sending the VT102 command for "cursor up", hopefully you'd see the start of the line change from `[1A[1K$<3>[K$<3>` to whatever the correct pseudoterminal command is.